.pre-commit-config.yaml

```
@@ -18,29 +18,29 @@ repos:
         exclude: \.md$
 
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.21.2
+    rev: 75992aaa40730136014f34227e0135f63fc951b4  # v3.21.2
     hooks:
     -   id: pyupgrade
```

YAMLlint (1.38.0)

[error] too many spaces after hyphen

(hyphens)